advanced-integration/v2/server/node/package.json

```diff
@@ -4,14 +4,16 @@
   "private": true,
   "type": "module",
   "dependencies": {
+    "@paypal/paypal-server-sdk": "^0.5.1",
+    "body-parser": "^1.20.3",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "node-fetch": "^3.3.2"
```

Comment:
Can we remove `node-fetch` since we are using `paypal-server-sdk`?

Reply:
removed